tests/test_orders.py

```diff
@@ -1,22 +1,21 @@
 import requests
+import requests_mock
+
 from woocommerce import API
 
 from mock_data.mock_order import order
+from app import woo_get
+
+BASE_URL = "https://www.anything.com"
 
-def test_list_orders(mocker):
-    url = "https://www.anything.com"
-    ck = "test_key"
-    cs = "test_secret"
-    wcapi = API(
```

Comment:
moved this code out, because it seems production code, so I rather tested the function from app